components/answers.js

```diff
@@ -0,0 +1,30 @@
+export const answers = [
+  'Maybe',
+  'Ask me later I am tired',
+  'Try to google it',
+  'Unpredictable outcome',
+  'Don’t count on it',
+  'It is certain',
+  'Almost impossible',
+  'Everythink is possible',
+  'My reply is no',
+  'Universe says no',
+  'Fortune is on your side',
+  'Signs point to yes',
+  'Will you dare?',
+  'Without a doubt',
+  'Yes',
+  'No',
+  'Force is with you!',
+  'Go for it!',
+  'Let it go',
+  'It will be fine, soon...',
+  'What your heart tells?',
+  'Leave it for later',
+  'Ask a human for advice!',
+  'Do not overthink',
+  'I am just a ball',
+  'Flip a coin',
+  'Explore it deeper',
+  'Take your chances',
```

Comment:
Haha! So this is how they randomise answers that will work for any questions in eight balls. Smart that it is done in single function and super simple to understand!